vertical-pod-autoscaler/enhancements/8515-support-custom-request-to-limit-ratio/README.md

```diff
@@ -0,0 +1,235 @@
+# AEP-8515: Add support for setting a custom request-to-limit ratio at the VPA object level
+
+<!-- toc -->
+- [Summary](#summary)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+    - [API Changes](#api-changes)
+    - [Behavior](#behavior)
+      - [Current behavior of VPA 1.4.2](#current-behavior-of-vpa-142)
+      - [Proposed feature behavior](#proposed-feature-behavior)
+    - [Notes/Constraints/Caveats](#notesconstraintscaveats)
+    - [Validation](#validation)
+      - [Static Validation via CRD Rules](#static-validation-via-crd-rules)
+      - [Dynamic Validation via Admission Controller](#dynamic-validation-via-admission-controller)
+    - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+      - [Enabling or Disabling the Feature in a Live Cluster](#enabling-or-disabling-the-feature-in-a-live-cluster)
+      - [When Enabled](#when-enabled)
+      - [When Disabled](#when-disabled)
+    - [Kubernetes Version Compatibility](#kubernetes-version-compatibility)
+    - [Test Plan](#test-plan)
+      - [E2E](#e2e)
+    - [Examples](#examples)
+  - [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+Currently, when VPA is configured to set both requests and limits automatically (i.e., `controlledValues` is set to `RequestsAndLimits` in the VerticalPodAutoscaler CRD), the limit is adjusted proportionally based on the initial request-to-limit ratio defined in workload API objects such as Deployments or StatefulSets - [Ref](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/docs/examples.md#keeping-limit-proportional-to-request).
```

Comment:
Nit: technically it's not the workload API that defines this, it's the Pod.

Reply:
addressed in a commit